Code/Tools/ProjectManager/Source/GemRepo/GemRepoModel.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QStandardItemModel>
+#include <GemRepo/GemRepoInfo.h>
+#endif
+
+QT_FORWARD_DECLARE_CLASS(QItemSelectionModel)
+
+namespace O3DE::ProjectManager
+{
+    class GemRepoModel
+        : public QStandardItemModel
+    {
+        Q_OBJECT // AUTOMOC
+
+    public:
+        explicit GemRepoModel(QObject* parent = nullptr);
+        QItemSelectionModel* GetSelectionModel() const;
+
+        void AddGemRepo(const GemRepoInfo& gemInfo);
+        void Clear();
+
+        QModelIndex FindIndexByNameString(const QString& nameString) const;
```

Comment:
looks like this is unused, may be able to remove it and the m_nameToIndexMap - unless you think you'll need it later?